zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/schema/elasticsearch/ElasticsearchEngineClient.java

```diff
@@ -84,6 +84,15 @@ public void createIndexTemplate(
       client.indices().putIndexTemplate(request);
       LOG.debug("Template [{}] was successfully created", templateDescriptor.getTemplateName());
     } catch (final IOException | ElasticsearchException e) {
+      if (e.getMessage()
+          .contains(
+              String.format(
+                  "index template [%s] already exists", templateDescriptor.getTemplateName()))) {
+        LOG.warn(
+            "Did not create index template [{}] as it already exists",
+            templateDescriptor.getTemplateName());
```

Comment:
💭 I'm still not sure whether this is really worth a warning. It is sort of expected that if I run Camunada with three partitions this will happen as we have no synchronization of who does the schema creation.

Reply:
🔧 I would like to see a clarification comment, why we think it is expected.

Reply:
I'd be fine with a debug as well